tools/tf2openapi/generator/generate.go

```diff
@@ -4,7 +4,12 @@ import (
 	pb "github.com/kubeflow/kfserving/tools/tf2openapi/generated/protobuf"
 )
 
-func GenerateOpenAPI(model *pb.SavedModel) (string, error) {
+const defaultSigDefKey = "serving_default"
```

Comment:
Do you mind defining a const block:

const (

)

Might be worth making it capitalized so you can test more easily.

Reply:
done